content/BCBestPractices/Readability/keyword-pairs-indentation/index.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Keyword Pairs - Indentation"
+weight = 730
++++
+
+<_Created by Microsoft, Described by waldo_\>
+
+## Description
+The `if..then` pair, `while..do` pair, and `for..do` pair must appear on the same line or the same level of indentation. 
+
+## Bad code
+
+```al
+    if (x = y) and
+    (a = b) then
+```
+
+## Good code
+
+```al
+    if (x = y) and
+    (a = b) 
+    then
+```
```

Comment:
Usually, I let the other line with the extra condition also indent. Shouldn't be part of the guideline?
```al
    if (x = y) and
       (a = b) 
    then
```